src/Pages/ProfilePage/ProfilePage.js

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
-import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
-import { useTranslation } from 'react-i18next';
+import React from './node_modules/react';
+import { MDBContainer, MDBRow, MDBCol } from './node_modules/mdbreact';
+import { useTranslation } from './node_modules/react-i18next';
```

Comment:
there is this extra `./node_modules/` in all your code and this is bringing an errors and you need to remove it

```suggestion
import React from 'react';
import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
import { useTranslation } from 'react-i18next';
```